dependencies.rosinstall

```diff
@@ -10,3 +10,7 @@
     local-name: opw_kinematics
     uri: https://github.com/Jmeyer1292/opw_kinematics.git
     version: master
+- git:
+    local-name: ifopt
+    uri: https://github.com/mpowelson/ifopt.git
+    version: feature/cost_cnt_getters
```

Comment:
Have you received any feed back on this change?

Reply:
I haven't submitted it yet. I was wanting to make sure there weren't any other changes we needed before I inundated him with PRs. I can go ahead and submit it though unless you know of anything else.

Reply:
He merged it over the weekend. I'll change this